### PR TITLE
Added support for attachCIDs argument - EmailAskUser

### PR DESCRIPTION
**Pull request by sapirshuker**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-31660)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by GuyAfik — APPROVED** [review]
🥇

**Comment by GuyAfik** [comment]
@sapirshuker please update PR's description to something more clear

File: Packs/CommonScripts/ReleaseNotes/1_13_19.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### EmailAskUser
+
+- Added support for ***attachCIDs*** argument in order to embed attachments within the actual email.
```
Comment (eyalpalo) [inline Packs/CommonScripts/ReleaseNotes/1_13_19.md:6]:
```suggestion
- Added support for ***attachCIDs*** argument to embed attachments within the email.
```

**Review by eyalpalo — APPROVED** [review]
Nice job with that!

**Comment by sapirshuker** [comment]
Validation failed only on: 
```
[2024-01-04 12:01:21,934] [ERROR] Packs/CommonScripts/Scripts/EmailAskUser/EmailAskUser.yml: [CJ105] - Either EmailAskUser does not have any test playbooks or that all test playbooks in this pack are currently skipped, and there is no unittests file to be found.
Please create a test playbook or un-skip at least one of the relevant test playbooks.
 You can un-skip a playbook by deleting the line relevant to one of the test playbooks from the 'skipped_tests' section inside the conf.json file and deal with the matching issue,
  or create a new active test playbook and add the id to the 'tests' field in the yml.
```
@dantavori can you please force merge?

**Pull request merged**